src/types/multiModel.js

```diff
@@ -15,14 +13,12 @@ class MultiModel extends BaseType {
         this.options = options || {};
         this.map = this.options.map;
         this.Model = Model;
-        this.requestCache = new RequestCache();
         this._listeners = {};
-        this._models = {};
     }
 
     _instantiateModel (id: ?string): Object {
         const { Model } = this;
-        const model = new Model(this._parent, this._parent, id, this.options.modelOptions);
+        const model = new Model(this._parent, id, this.options.modelOptions);
```

Comment:
Hmmmm. I have concerns about making and storing models in MultiModel. We should only instantiate new models on OwnsMany.